### ISSUE TITLE
Reminder: handle enable/disable RTTI in windows.toolchain.cmake

### ISSUE THREAD

**Issue by floooh** [floooh/fips#86]
...need to handle FIPS_RTTI setting in windows toolchain file.

**Comment by fungos** [comment]
Looks like FIPS_EXCEPTIONS needs some handling on windows too, right?

**Comment by floooh** [comment]
Yeah, the visual studio exception-related flags are utterly confusing. I'll be mainly on Windows in the next few weeks, may be I'll get around to look into it.
